lib/scripts/default.nix

```
@@ -11,7 +11,7 @@ let
       owner = "IObundle";
       repo = "py2hwsw";
       rev = py2hwsw_commit;
-      sha256 = "72SBftfmGKldDt7cFNz4aIB2hFytWtO6tivJKMaTlNA=";  # Replace with the actual SHA256 hash.
+      sha256 ="sPYxs4lHzHh+vfwxoSKY0GyBE0k521ruSn14KLT1Eeg=";  # Replace with the actual SHA256 hash.
```

We should update the corresponding sha265 hash as well:
```suggestion
      sha256 ="wxk/srPoqzwnhW8mx/cGZpS66AueSE+5cw5c2xxUyJQ=";  # Replace with the actual SHA256 hash.
```